client/lib/signup/step-actions.js

```diff
@@ -53,74 +137,31 @@ function setThemeOnSite( callback, { siteSlug }, { themeSlug } ) {
 }
 
 module.exports = {
-	addDomainItemsToCart( callback, dependencies, { domainItem, googleAppsCartItem, isPurchasingItem, siteUrl, themeSlug, themeItem } ) {
-		wpcom.undocumented().sitesNew( {
-			blog_name: siteUrl,
-			blog_title: siteUrl,
-			options: {
-				theme: dependencies.theme
-			},
-			validate: false,
-			find_available_url: isPurchasingItem
-		}, function( error, response ) {
-			if ( error ) {
-				return callback( error );
-			}
+	addDomainItemsToCart: addDomainItemsToCart,
```

Comment:
I'll take care of that during hack day (now you know what I'm gonna do :smile: ).